### DISCUSSION TITLE
building Interface

### DISCUSSION THREAD

**Discussion by clem-ens94** [cmcgerty/Sofar2mqtt#83] [General]
Hello,
I'm a totally newbie in HA, Sofar2MQTT und solar inverters in general. I already set up HA and some things but I never will be able to build my own interface for Sofar2MQTT. Is there any way to buy something like this?

**Comment by cliffdude** [comment]
Where are you located? It really is not that hard to build though. That
said there are some ESP8266 boards with an RS485 interface on aliexpress.
That is all you need and then flash it with Sofar2Mqtt. The Oled display is
not necessary

A terça, 10/09/2024, 08:52, clem-ens94 ***@***.***> escreveu:

**Reply by clem-ens94** [reply]
I‘m from Germany. My Problem is, I have only have one Hand, so soldering etc. is really Hard for me and I don‘t wanna try to Tell my wife how she has to do :-D Do you have a link to Ali for me?
